### PR TITLE
Clarify the interface documentation an migration.

### LINKED ISSUE #2524: Update the migration notes for an old cgxp interface
- update the changelog file
- remove the automation about the index.html - viewer.json
- document everything related to the interface
  - all the file needed for an ngeo/cgxp interface
  - the make variable: CGXP_INTERFACES/NGEO_INTERFACES
  - the vars variable: default_interface

### PR DESCRIPTION
**Pull request by sbrunner**
fix #2524

### CODE REVIEW

File: doc/integrator/interface.rst [hunk 1]
```diff
@@ -28,3 +50,24 @@ also used to create the route path.
 
 ``interface_type`` may be either ``INTERFACE_TYPE_CGXP``, ``INTERFACE_TYPE_NGEO`` or
 ``INTERFACE_TYPE_NGEO_CATALOGUE``. Constants available in ``c2cgeoportal``.
+
+
+Makefile
+--------
+
+In the makefile (``<package>.mk``) you can define the variable ``CGXP_INTERFACES`` and
+``NGEO_INTERFACES`` that should contains the interfaces lists.
```
Comment (gberaudo) [inline doc/integrator/interface.rst]: contain

File: doc/integrator/interface.rst [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
+
+Files
+-----
+
+A CGXP interface requires those files::
+
+* ``<package>/template/<interface name>.html``
+* ``<package>/template/<interface name>.js``
+
+And a ngeo interface requires those files::
```
Comment (gberaudo) [inline doc/integrator/interface.rst]: s/a/an/

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 3]
```diff
@@ -754,7 +754,7 @@ $(APP_OUTPUT_DIR)/%.json: $(PACKAGE)/locale/%/LC_MESSAGES/$(PACKAGE)-client.po \
 	mkdir -p $(dir $@)
 	node $(COMPILE_CATALOG) .build/locale/$*/LC_MESSAGES/gmf.po $< > $@
 
-$(APP_OUTPUT_DIR)/angular-locale_%.js: node_modules/ngeo/github_versions
+$(APP_OUTPUT_DIR)/angular-locale_%.js: node
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: ??? => will be a fixed typo :-)

**Comment by gberaudo** [comment]
LGTM

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**